### ISSUE TITLE
Add a tracker for products

### ISSUE THREAD

**Issue by amenk** [pug-more/mageploy#26]
How much effort would it be to add a tracker for products?

**Comment by aleron75** [comment]
I never implemented it because I always use Import/Export to sync catalog. 
Anyway I think the effort would not be irrelevant, considering also that media uploads are involved.
What are the cons of delegating product sync to Import/Export in your opinion?
Thank you
Alessandro

**Comment by amenk** [comment]
Maybe to lazy for export/import but it is an option. I get the point with the media files. But I our case we do not have media. It just would be nice to have all in one place (mageploy files).

**Comment by aleron75** [comment]
I agree; actually products have a clear UUID (the SKU) so it shouldn't be a problem from that point of view. The main problem could be the complexity of data structure that should be serialized/unserialized: it would require a lot of test with different product types.
Sincerely,
Alessandro

**Comment by electric** [comment]
Hi there - I've recently discovered Mageploy and like the functionality. I see this project has potentially stalled and would be interested in any further developments, particularly around a tracker for products. Any updates?

Thanks

**Comment by aleron75** [comment]
Hello @electric, I don't have plans for future development for Mageploy since it's bound to Magento 1.

It would be a good thing writing a new version for Magento 2 but at the moment I'm spending my limited spare time on Magento 2 itself and furthermore, I'm a bit stuck in analysis paralysis.

Cheers,
-Alessandro
